### DISCUSSION TITLE
Translation of the app

### DISCUSSION THREAD

**Discussion by only1avetrill** [nukeop/nuclear#1239] [Ideas]
I know that some users from my country like to use apps on native Belarusian language. I can translate it and also spread it here, in Belarus. If you need it, add this language to Crowdin and i'll start to work.

**Comment by nukeop** [comment]
You can translate here: https://crowdin.com/project/nuclear

I added Belarusian to the list of languages. Let me know once you've translated anything, I will merge your changes and make it available in the settings.

**Reply by only1avetrill** [reply]
> You can translate here: https://crowdin.com/project/nuclear
> 
> I added Belarusian to the list of languages. Let me know once you've translated anything, I will merge your changes and make it available in the settings.

Thanks you so much! Started translation and already added some strings, check it out

**Reply by nukeop** [reply]
Great, thank you. If you need any help (like missing context) please leave a comment on Crowdin.

**Comment by abiramcodes** [comment]
Hello @nukeop , 
I would like to translate to one of the oldest / native language _**TAMIL.**_ 

Could you please add the language on Crowdin ?

Thanks,
Abiram

**Reply by nukeop** [reply]
Ok, I added Tamil.
